language-plutus-core/src/Language/PlutusCore.hs

```diff
@@ -95,6 +95,7 @@ module Language.PlutusCore
     -- * Evaluation
     , EvaluationResultF (EvaluationSuccess, EvaluationFailure)
     , EvaluationResult
+    , runCk
```

Comment:
We discussed this before. Not exporting `runCk`/`evaluateCk` from `PlutusCore.hs` is very deliberate and approved by Manuel.